shell/common/animator.cc

```diff
@@ -121,12 +111,8 @@ void Animator::BeginFrame(
   const fml::TimePoint frame_target_time =
       frame_timings_recorder_->GetVsyncTargetTime();
   dart_frame_deadline_ = FxlToDartOrEarlier(frame_target_time);
-  {
-    TRACE_EVENT2("flutter", "Framework Workload", "mode", "basic", "frame",
-                 FrameParity());
```

Comment:
This (and FrameParity) were used by a no longer functioning view in the observatory. See https://github.com/dart-lang/sdk/blob/main/runtime/observatory_2/web/timeline.js#L158 and related lines in there, which also refer to now-deleted timeline events.

Reply:
Also, this is redundant with Animator::BeginFrame, which has the frame number in it and is easily detected as even or odd.